src/base/utils.ml

```diff
@@ -1,48 +1,474 @@
 open Core_kernel
-
-(** Alias [Cvar] to avoid shadowing. *)
+module Types0 = Types
```

Comment:
Searching the file for `Types0` returns no occurrence. Is this alias necessary?

Reply:
Will remove.